workflows/phylogenetics/wf_augur.wdl

```diff
@@ -150,8 +152,8 @@ workflow augur {
                             augur_clades.clade_assignments_json,
                             augur_traits.traits_assignments_json]),
         build_name = build_name,
-        lat_longs_tsv = select_first([sc2_defaults.lat_longs_tsv, flu_defaults.lat_longs_tsv, mpxv_defaults.lat_longs_tsv, lat_longs_tsv]),
-        auspice_config = select_first([sc2_defaults.auspice_config, flu_defaults.auspice_config, mpxv_defaults.auspice_config, auspice_config])
+        lat_longs_tsv = select_first([organism_parameters.augur_lat_longs_tsv, sc2_defaults.lat_longs_tsv, lat_longs_tsv]),
+        auspice_config = select_first([organism_parameters.augur_auspice_config, sc2_defaults.auspice_config, auspice_config])
```

Comment:
change priority - user-provided inputs to beginning, then SC2_defaults, then organism_parameters

Reply:
Done